### ISSUE TITLE
Since version 300 it is impossible to remove installed applications

### ISSUE THREAD

**Issue by Eusino** [rashevskyv/dbi#42]
since version 300, it is impossible to remove installed applications,
one by one or several selected.
The only menu that appears is Reset required version

**Comment by rashevskyv** [comment]
Please read changelogs and readme of the app. Use plus button gor context
menu

ср, 7 июл. 2021 г., 06:57 Eusino ***@***.***>:

**Comment by Eusino** [comment]
Sorry!
I've already seen what happens,
Before version 300 it was possible to remove the files that are installed from using game cards (these are 0 bytes)
Now those files cannot be deleted since pressing + only allows you to reset required version ...
When I press + in an installed game that is different from 0 bytes (actually installed on the console), all the other options appear.

This means that now it is not possible to delete files from cartridges or xci. (Without installing the game on the console)
Although the option to delete selected appears if one of the files that is selected is actually installed on the console (different from 0 bytes)
In summary now it does not allow to uninstall game entries from cartridges

**Comment by rashevskyv** [comment]
You cannot uninstall game witch cartridge is inserted. Remove or unmount cartrige first

**Comment by Eusino** [comment]
I do not mean the inserted cartridges if not the file they leave
once these are out of the console

**Comment by rashevskyv** [comment]
Confirmed this issue

**Comment by duckbill007** [comment]
Fixed in 308

**Comment by Eusino** [comment]
Thanks you!!!

**Comment by rashevskyv** [comment]
https://github.com/rashevskyv/dbi/releases/tag/308
